corehq/warehouse/tests/test_dimensions.py

```diff
@@ -203,14 +203,14 @@ def test_location_dim(self):
         home_location = LocationDim.objects.filter(name='Home').first()
 
         self.assertEqual(
-            home_location.location_level_2,
+            home_location.location_level_0,
             LocationDim.objects.filter(name='Illinois').first().sql_location_id,
         )
         self.assertEqual(
             home_location.location_level_1,
             LocationDim.objects.filter(name='Naperville').first().sql_location_id,
         )
-        self.assertEqual(home_location.location_level_0, home_location.sql_location_id)
+        self.assertEqual(home_location.location_level_2, home_location.sql_location_id)
```

Comment:
could we validate all the location ids for just one record in one of these tests (including the null ones)? could help to make sure nothing breaks if changes ever get made to the routine